cli/src/main/kotlin/com/malinskiy/marathon/cli/config/deserialize/TestFilterDeserializer.kt

```diff
@@ -8,12 +8,7 @@ import com.fasterxml.jackson.databind.deser.std.StdDeserializer
 import com.fasterxml.jackson.databind.node.ObjectNode
 import com.fasterxml.jackson.module.kotlin.treeToValue
 import com.malinskiy.marathon.exceptions.ConfigurationException
-import com.malinskiy.marathon.execution.AnnotationFilter
-import com.malinskiy.marathon.execution.FullyQualifiedClassnameFilter
-import com.malinskiy.marathon.execution.SimpleClassnameFilter
-import com.malinskiy.marathon.execution.TestFilter
-import com.malinskiy.marathon.execution.TestMethodFilter
-import com.malinskiy.marathon.execution.TestPackageFilter
+import com.malinskiy.marathon.execution.*
```

Comment:
We don't use wildcard imports in the project. Please correct this in the IDE :) and repush

Reply:
done